### PR TITLE
[enhancement](transaction) Reduce hold writeLock time for DatabaseTransactionMgr to clear transaction

### PR DESCRIPTION
**Pull request by caiconghui**
# Proposed changes

Issue Number: close #xxx

## Problem summary

make clear transaction operation log more light to reduce hold writeLock time

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by caiconghui** [comment]
run all build

**Comment by caiconghui** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java [hunk 1]
```diff
@@ -742,6 +742,37 @@ public void replayBatchRemoveTransaction(List<Long> txnIds) {
         }
     }
 
+    public void replayBatchRemoveTransaction(BatchRemoveTransactionsOperationV2 operation) {
+        writeLock();
+        try {
+            int numOfClearedTransaction = 0;
+            if (operation.getLatestTxnIdForShort() != -1) {
+                while (!finalStatusTransactionStateDequeShort.isEmpty()) {
+                    TransactionState transactionState = finalStatusTransactionStateDequeShort.pop();
```
Comment (wangbo) [inline fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java:751]: In old version,  when execute ```replayBatchRemoveTransaction```, txnId could be removed when txnId locates in head of deque.
So in V2, how to ensure the consistency between latestTxnIdForShort/latestTxnIdForLong and finalStatusTransactionStateDequeShort/finalStatusTransactionStateDequeLong.
Is there the case that latestTxnIdForShort not in finalStatusTransactionStateDequeShort?
How about add a check here?

Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java:751]: only when is there a bug, never encounter the unconsistent case here

**Review by wangbo — DISMISSED** [review]
LGTM

**Comment by caiconghui** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**